lighthouse-cli/manual-chrome-launcher.js

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+
+'use strict'
+
+require('./compiled-check.js')('chrome-launcher.js');
+
+const args = process.argv.slice(2);
+
+let chromeArgs;
+let startingUrl;
+
+if (args.length) {
+  // If the first argument starts with -- assume all are chrome args.
+  if (args[0].startsWith('--')) {
```

Comment:
I meant it should work already with launch-chrome.sh, rather. whoops.

i dont feel strongly that we need to support opening multiple via this, so w/e.
what you did in the latest commit sgtm